extensions/wikia/TemplateClassification/scripts/TemplateClassificationInEdit.js

```diff
@@ -11,10 +11,12 @@ define('TemplateClassificationInEdit',
 	function ($, mw, tracker, templateClassificationModal) {
 		'use strict';
 
-		var $editFormHiddenTypeField;
+		var $editFormHiddenTypeFieldCurrent,
+			$editFormHiddenTypeFieldNew;
 
 		function init() {
-			$editFormHiddenTypeField = $('#editform').find('[name=templateClassificationType]');
+			$editFormHiddenTypeFieldCurrent = $('#editform').find('[name=templateClassificationTypeCurrent]');
+			$editFormHiddenTypeFieldNew = $('#editform').find('[name=templateClassificationTypeNew]');
```

Comment:
We can cache `$('#editform')` in var. I think that filtering these fields by class name will be more performant